javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java

```diff
@@ -154,6 +155,16 @@ private List<ClientMethod> createClientMethods(Operation operation) {
             }
         }
 
+        // map externalDocs property
+        if (operation.getExternalDocs() != null) {
+            MethodDocumentation methodDocumentation = new MethodDocumentation.Builder()
+                    .description(operation.getExternalDocs().getDescription())
+                    .url(operation.getExternalDocs().getUrl())
+                    .extensions(operation.getExternalDocs().getExtensions())
```

Comment:
why you need the extensions?

Reply:
I don't need it for display currently, but set here to keep all the information. Can be deleted as well.